src/app/core/services/logging.service.ts

```diff
@@ -2,6 +2,8 @@ import { Injectable } from '@angular/core';
 import { ElectronService } from './electron.service';
 import { ElectronLog } from 'electron-log';
 
+const appSetting = require('../../../../package.json');
```

Comment:
Just a query, If we are exporting `currentVersion` as a `const` in `application.service.ts` would it work if we just import the constant directly to the `logging.service.ts`? That way we don't have to import `package.json` twice and can just use the version number below? Would this work?

Reply:
It actually results in a circular dependency unfortunately due to the following: 

`ApplicationService` -> `GithubService` -> `ErrorHandlingService` -> `LoggingService` -> `ApplicationService`. As a result, I have decided note to export `currentVersion` from `ApplicationService` 

Is there actually a better way of handling this though, like maybe via a constant somewhere else? o.O

Reply:
Since we have this dependency issue, we can store the version number in the `AppConfig` instead.
@seanlowjk  can you do this, and then tweak all parts of the codebase that require the version number, such that they import it from `AppConfig` ?

The changes will be needed in:
- ApplicationService
- IssueService (records version number in a HTML comment in the issue body)
- AuthService (displays version number in page title)
- LoggingService (for this PR)
- HeaderComponent (this PR)

Reply:
Since we have this dependency issue, we can store the version number in the `AppConfig` instead.
@seanlowjk  can you do this, and then tweak all parts of the codebase that require the version number, such that they import it from `AppConfig` ?

The changes will be needed in:
- ApplicationService
- IssueService (records version number in a HTML comment in the issue body)
- AuthService (displays version number in page title)
- LoggingService (for this PR)
- HeaderComponent (this PR)

Reply:
Alright, before I continue! Just to double confirm, the above 3 tasks are to be done in another PR right? :O @anubh-v

Reply:
Sure, let's limit this PR to changes in LoggingService and HeaderComponent.
Can make another PR to standardise the method of accessing version number from `AppConfig` in the rest of the codebase.

Reply:
Alright!